modin/pandas/base.py

```diff
@@ -2791,59 +2791,22 @@ def sort_values(
              A sorted DataFrame.
         """
         axis = self._get_axis_number(axis)
-        if not is_list_like(by):
-            by = [by]
-        # Currently, sort_values will just reindex based on the sorted values.
-        # TODO create a more efficient way to sort
-        ErrorMessage.default_to_pandas("sort_values")
+        inplace = validate_bool_kwarg(inplace, "inplace")
         if axis == 0:
-            broadcast_value_dict = {col: self[col]._to_pandas() for col in by}
-            # Index may contain duplicates
-            broadcast_values1 = pandas.DataFrame(broadcast_value_dict, index=self.index)
-            # Index without duplicates
-            broadcast_values2 = pandas.DataFrame(broadcast_value_dict)
-            broadcast_values2 = broadcast_values2.reset_index(drop=True)
-            # Index may contain duplicates
-            new_index1 = broadcast_values1.sort_values(
-                by=by,
-                axis=axis,
-                ascending=ascending,
-                kind=kind,
-                na_position=na_position,
-            ).index
-            # Index without duplicates
-            new_index2 = broadcast_values2.sort_values(
-                by=by,
-                axis=axis,
+            result = self._query_compiler.sort_rows_by_column_values(
+                by,
                 ascending=ascending,
-                kind=kind,
```

Comment:
It seems that you doesn't using `kind` parameter. Why so?

Reply:
If you don't mind, let's delegate responsibility for processing `kind` parameter to query compilers.